src/nvim/generators/gen_ex_cmds.lua

```diff
@@ -21,24 +21,32 @@ local a_to_z = byte_z - byte_a + 1
 
 -- Table giving the index of the first command in cmdnames[] to lookup
 -- based on the first letter of a command.
-local cmdidxs1_out = string.format([[
+local cmdidxs1_out = string.format(
+  [[
 static const uint16_t cmdidxs1[%u] = {
-]], a_to_z)
+]],
```

Comment:
Not a fan of these changes to `[[...]]`, but the rest of changes are largely an improvement.